monitoring/dashboard.json

```diff
@@ -2348,7 +2348,7 @@
     ]
   },
   "timezone": "",
-  "title": "CloudServer",
+  "title": "S3 service",
```

Comment:
Should we use the word 'service'?

Reply:
This is what was asked by @rahulreddy, after a few variations (S3 Api, S3...)